warden/app/wasm-interop/custom_msg.go

```diff
@@ -0,0 +1,47 @@
+package wasm_interop
```

Comment:
is this replacing `custom_msg/custom_msg.go`? I don't see any file deleted by this PR. Do we need both?